sdk/servicebus/azure-servicebus/tests/async_tests/test_sb_client_async.py

```diff
@@ -414,14 +453,16 @@ async def test_client_named_key_credential_async(self,
             async with client.get_queue_sender(servicebus_queue.name) as sender:
                 await sender.send_messages(ServiceBusMessage("foo"))
 
-    async def test_backoff_fixed_retry(self):
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    def test_backoff_fixed_retry(self, uamqp_transport):
         client = ServiceBusClient(
             'fake.host.com',
             'fake_eh',
-            retry_mode='fixed'
+            retry_mode='fixed',
+            uamqp_transport=uamqp_transport
         )
         # queue sender
-        sender = await client.get_queue_sender('fake_name')
+        sender = client.get_queue_sender('fake_name')
```

Comment:
Why did we drop the `await`s here? Or was this test always broken?

Reply:
this test was broken before and wasn't getting picked up, b/c the test was defined as async without an asyncio decorator.